src/silverlabnwb/header.py

```diff
@@ -253,3 +261,10 @@ def determine_trial_times(self):
                 end = None  # determine final 'end' later from speed data
             trial_times.append((start, end))
         return trial_times
+
+    def get_subject_info(self):
+        return "\n".join(
```

Comment:
I wonder whether `". ".join(...` is better (to have everything in a single line. It could get quite long if we have a lot of information though.